### ISSUE TITLE
Unexpected keyword argument 'fallback'

### ISSUE THREAD

**Issue by Skarzee** [di/bump#6]
Hey first off thanks for the tool, found it very useful over the years. Thanks for picking it up and maintaining it!

I've found an issue with this when running with 2.7.15 
```
~/Work/vault-provisioner
vp ❯ python --version
Python 2.7.15

~/Work/vault-provisioner
vp ❯ bump
Traceback (most recent call last):
  File "/Users/Damien/.virtualenvs/vp/bin/bump", line 11, in <module>
    sys.exit(main())
  File "/Users/Damien/.virtualenvs/vp/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/Users/Damien/.virtualenvs/vp/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/Users/Damien/.virtualenvs/vp/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/Damien/.virtualenvs/vp/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/Users/Damien/.virtualenvs/vp/lib/python2.7/site-packages/bump.py", line 126, in main
    major = major or config.getboolean('bump', 'major', fallback=False)
TypeError: getboolean() got an unexpected keyword argument 'fallback'
```

I've had a ponder at the code but no ideas right now

**Comment by Skarzee** [comment]
Looking further into this, it was because within my venv ConfigParser wasn't installed within my venv:

```~/Work/vault-provisioner
vp ❯ pip list
Package     Version
----------- ---------
bump        1.1.0
certifi     2018.8.13
chardet     3.0.4
click       6.7
first       2.0.1
hvac        0.6.4
idna        2.7
packaging   18.0
pip         18.1
ply         3.10
provisioner 0.1
pyhcl       0.3.10
pyparsing   2.3.0
PyYAML      3.13
requests    2.19.1
setuptools  40.5.0
six         1.11.0
urllib3     1.23
wheel       0.32.2

~/Work/vault-provisioner
vp ❯ pip install ConfigParser
Collecting ConfigParser
  Downloading https://files.pythonhosted.org/packages/7c/69/c2ce7e91c89dc073eb1aa74c0621c3eefbffe8216b3f9af9d3885265c01c/configparser-3.5.0.tar.gz
Building wheels for collected packages: ConfigParser
  Running setup.py bdist_wheel for ConfigParser ... done
  Stored in directory: /Users/Damien/Library/Caches/pip/wheels/a3/61/79/424ef897a2f3b14684a7de5d89e8600b460b89663e6ce9d17c
Successfully built ConfigParser
Installing collected packages: ConfigParser
Successfully installed ConfigParser-3.5.0

~/Work/vault-provisioner
vp ❯ bump
0.2.1
```

Would it be a good idea to include a requirements.txt for this? We can probably close this issue

**Comment by di** [comment]
Hmm, `ConfigParser` is [in the standard library](https://docs.python.org/2/library/configparser.html) for Python 2.7, you shouldn't need to `pip install` it here.

**Comment by di** [comment]
Ah, I see, the `fallback` param is not available in the Python 2 standard library. You're right, this should have a dependency on the `configparser` package.

**Comment by di** [comment]
Thanks for the report! This is fixed in [`bump==1.1.1`](https://pypi.org/project/bump/1.1.1/).
